### DISCUSSION TITLE
How to limit Product Types to Sites/Site Groups?

### DISCUSSION THREAD

**Discussion by gopeter** [craftcms/commerce#3372] [Q&A]
Is there a way to define product types that are available for specific sites (or even better: site groups) only? Like it's already possible for sections:

<img width="1229" alt="image" src="https://github.com/craftcms/commerce/assets/1242960/6c9706ca-860b-4f45-8da9-049a8c47fe8a">

**Comment by lukeholder** [comment]
Not at the moment. Products types are available on all sites by default - you can turn of individual products in each site though.

Commerce 5 will introduce multi-store, and you can have products enabled and disabled in different stores (which sit under a site).

**Reply by gopeter** [reply]
OK great! Is there already a Beta to try this out? 😬

**Reply by gopeter** [reply]
Hi @lukeholder, I'm playing with the latest beta versions and I can't disable some product types for specific sites. Is this something that may come in the future?
